Source/WebCore/editing/Editing.cpp

```diff
@@ -574,6 +574,12 @@ Element* enclosingAnchorElement(const Position& p)
     return nullptr;
 }
 
+Element* enclosingAtomicNode(const Position& start)
+{
+    auto start = m_selectionToDelete.start();
```

Comment:
This line shouldn't be here.

Reply:
Should I delete it and add it within header 'Editing.h' as return value?

Reply:
No, this line isn't needed at all. There is no m_selectionToDelete defined here.

Reply:
Done.